h2o-algos/src/main/java/hex/glm/GLMMetricBuilder.java

```diff
@@ -27,6 +27,7 @@
   double residual_deviance;
   double null_devince;
   long _nobs;
+  double _sumLLH;
```

Comment:
follow the naming conventions, please rename to log_likelihood

Reply:
Done